mbedtls-sys/build/cmake.rs

```diff
@@ -19,6 +19,8 @@ impl super::BuildConfig {
         .define("GEN_FILES", "ON")
         // Prefer unix-style over Apple-style Python3 on macOS, required for the Github Actions CI
         .define("Python3_FIND_FRAMEWORK", "LAST")
+        // Ensure "lib" directory is used on all platforms
+        .define("LIB_INSTALL_DIR", "lib")
```

Comment:
This should be a constant that's used later on line 48.

Reply:
Good catch! I've created [PR378](https://github.com/fortanix/rust-mbedtls/pull/378) for this